src/amo/components/StaticPages/ReviewGuide.js

```diff
@@ -0,0 +1,117 @@
+import Helmet from 'react-helmet';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { compose } from 'redux';
+
+import Card from 'ui/components/Card';
+import translate from 'core/i18n/translate';
+import { sanitizeHTML } from 'core/utils';
+
+import 'amo/components/StaticPages/StaticPages.scss';
+
+
+export class ReviewGuideBase extends React.Component {
+  static propTypes = {
+    i18n: PropTypes.object.isRequired,
+  }
+
+  render() {
+    const { i18n } = this.props;
+
+    return (
+
+      <Card
+        className="StaticPage"
+        header={i18n.gettext('Review Guidelines')}
+      >
+        <Helmet>
+          <title>{i18n.gettext('Review Guidelines')}</title>
+        </Helmet>
+
+        <section id="review-guide">
+          <p>
+            {i18n.gettext(`Add-on reviews are a way for you to share your opinions
+              about the add-ons you’ve installed and used. Our review moderation
+              team reserves the right to refuse or remove any review that does not
+              comply with these guidelines.`)}
+          </p>
```

Comment:
I'd add a new paragraph that lists some more guidelines to get them out of list format:

```
<p>
  {i18n.gettext('Please only write reviews for add-ons you have personally used. If you like or dislike an add-on but don't want to leave a written review, you can rate an add-on by giving it a star rating (out of five)–but please don't leave one-word reviews like "Wonderful.", "bad", or "Slow!"')}
</p>
```